CHANGELOG.md

```diff
@@ -119,6 +119,9 @@ Changelog
   are passed to the QVM by `WavefunctionSimulator.run_and_measure()` and `WavefunctionSimulator.wavefunction()`.
   
 - `noise.estimate_assignment_probs()` now accepts a `QuantumComputer` instead of `QVMConnection`.
+
+- `get_qc` and `QPU` now both accept an `endpoint_id` argument which is used to engage
```

Comment:
Nit: "Both `get_qc` and `QPU` now accept..." reads easier IMO.

Reply:
agreed, fixed